clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListFilesetProperties.java

```diff
@@ -60,27 +60,28 @@ public ListFilesetProperties(
   @Override
   public void handle() {
     Fileset gFileset = null;
-    NameIdentifier name = null;
+    
     try {
-      name = NameIdentifier.of(schema, fileset);
-    } catch (Exception exp) {
-      exitWithError("Invalid schema or fileset name: " + exp.getMessage());
-      return;
-    }
-    try (GravitinoClient client = buildClient(metalake)) {
-      gFileset = client.loadCatalog(catalog).asFilesetCatalog().loadFileset(name);
+        NameIdentifier name = NameIdentifier.of(schema, fileset);
+
+        try (GravitinoClient client = buildClient(metalake)) {
```

Comment:
as I mentione dbefore that may close the client before we can make other  needed calls